### DISCUSSION TITLE
how to restart zombienet from data of previous run?

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [paritytech/zombienet#1325] [Q&A] [answered]
I have run zombienet, got some data.
I stopped it.
I want to run it again from same storage, how can i do that?

**Comment by pepoviola** [comment, marked as answer]
Hi @dzmitry-lahoda, the option you have is to set the `directory` to use with the flag `-d` and past the previous _working directory_. For example, if you first  run zombienet with `-d /tmp/zombie-demo` and you stop the network the second time that you run with `-d /tmp/zombie-demo` will reuse the same dbs for the nodes.

Ping me if you have questions, thanks!!
